content/docs/state-and-lifecycle.md

```diff
@@ -57,9 +57,9 @@ setInterval(tick, 1000);
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/dpdoYR?editors=0010)
 
-However, it misses a crucial requirement: the fact that the `Clock` sets up a timer and updates the UI every second should be an implementation detail of the `Clock`.
+हालांकि, यह तथ्य कि `Clock` एक टाइमर सेट करना और हर सेकंड UI को अपडेट करना `Clock` का कार्यान्वयन विवरण होना चाहिए।
```

Comment:
revisit sentence again?

Reply:
"हालांकि, यह तथ्य कि `Clock` एक टाइमर सेट करना और हर सेकंड UI को अपडेट करना `Clock` का प्रशासन का विवरण होना चाहिए।" is this ok?

Reply:
हालांकि, यह एक महत्वपूर्ण आवश्यकता ko नज़रअंदाज़ karta hai. यह तथ्य कि Clock एक टाइमर सेट karta hai  और हर सेकंड UI को अपडेट करना Clock ks कार्यरूप विवरण hona chahiye
how's that?